src/data/actions/utils.js

```diff
@@ -11,54 +11,27 @@ const sortAlphaAsc = (gradeRowA, gradeRowB) => {
 };
 
 const headingMapper = (filterKey) => {
-  function all(entry) {
+  const filters = {
```

Comment:
I have chosen to remove the label from Redux, as I didn't want to tie the label that is displayed for each column into Redux, particularly since the Username column label is now an element and not just a string.

Since we use this state to determine what filters have been applied to the headings, I have changed it to an array of keys.

Reply:
I have also done some refactoring to make this function more DRY.